src/node_url.cc

```diff
@@ -1477,6 +1477,9 @@ void URL::Parse(const char* input,
           if (ch != '/') {
             continue;
           }
+        } else if (has_state_override && !(url->flags & URL_FLAGS_HAS_HOST)) {
+          url->flags |= URL_FLAGS_HAS_PATH;
```

Comment:
@addaleax @jasnell @XadillaX I had to add this line to make the tests pass. Since the code changed, could this please get another review before I add the `author-ready` label please?

Reply:
@addaleax @jasnell @XadillaX ^ 🙏

Reply:
Reviewed.